pom.xml

```diff
@@ -129,6 +129,7 @@
     <java.min.version>${compileSource}</java.min.version>
     <maven.min.version>3.3.3</maven.min.version>
     <rename.offset>org.apache.hbase.thirdparty</rename.offset>
+    <rename.offset.dir>org/apache/hbase/thirdparty</rename.offset.dir>
```

Comment:
Either a note that says it must match the package above or create it using the build-helper plugin to replace "." with "/"